src/sentry/tagstore/snuba/backend.py

```diff
@@ -242,12 +242,14 @@ def get_group_tag_keys_and_top_values(self, project_id, group_id, environment_id
             'issue': [group_id],
         }
         aggregations = [
-            ['topK(10)', 'tags.value', 'top'],
             ['count()', '', 'count'],
-            ['uniq', 'tags.key', 'uniq'],
+            ['topK(10)', 'tags_value', 'top'],
+            ['uniq', 'tags_value', 'uniq'],
         ]
-        results = snuba.query(start, end, ['tags.key'], None, filters,
-                              aggregations, arrayjoin='tags')
+        conditions = [
+            ['tags_value', 'IS NOT NULL', None],
```

Comment:
This looked weird to me at first, is this equivalent to `['tags_value', 'IS NOT', 'NULL']`?

Reply:
yeah, the schema still enforces the 3rd element is there, even for unary operators like this. I can probably change it, but need to make sure the whole AND/OR condition nesting doesn't rely on it somehow.